### PR TITLE
feat: new template in this repo

### PR DESCRIPTION
**Pull request by jeremylgn**
Self explanatory.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/user-story-template.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+labels: "Issue type : User story"
+assignees: ""
+---
+
+# Feature: [Name]
+
+> **As a** [user concerned by the story],
+> **I want** to [goal of the story]
+> **So that** [reason for the story]
+## Acceptance criteria
```
Comment (ccamel) [inline .github/ISSUE_TEMPLATE/user-story-template.md:14]:
```suggestion

## Acceptance criteria
```

File: .github/ISSUE_TEMPLATE/user-story-template.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+1. Constraint 1;
+1. Constraint 2;
+1. Constraint 3.
+
+## Resources
+
+### Mockups
+
+Desktop
```
Comment (ccamel) [inline .github/ISSUE_TEMPLATE/user-story-template.md:34]:
```suggestion
#### Desktop
```

File: .github/ISSUE_TEMPLATE/user-story-template.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+Mobile
```
Comment (ccamel) [inline .github/ISSUE_TEMPLATE/user-story-template.md:36]:
```suggestion
#### Mobile
```

File: .github/ISSUE_TEMPLATE/user-story-template.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+### Environment
+
+- staging URL: https://app.staging.okp4.network/
```
Comment (ccamel) [inline .github/ISSUE_TEMPLATE/user-story-template.md:40]:
```suggestion
- staging URL: <https://app.staging.okp4.network/>
```

File: .github/ISSUE_TEMPLATE/user-story-template.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+**And** I am using a *desktop* device,
+**When** [I do A]
+**Then** [B should happen]
+**And** [C should happen]
+
+## Requirements
+
+1. Constraint 1;
+1. Constraint 2;
+1. Constraint 3.
```
Comment (ccamel) [inline .github/ISSUE_TEMPLATE/user-story-template.md:28]:
```suggestion
1. [Requirement 1]
1. [Requirement 2]
1. [Requirement 3]
```

**Review by ccamel — CHANGES_REQUESTED** [review]
Great. Thanks.

I'd suggest some minor style improvements.

File: .github/ISSUE_TEMPLATE/user-story-template.md (hunk 1 above)
Comment (jeremylgn) [reply .github/ISSUE_TEMPLATE/user-story-template.md:15]: Done :heavy_check_mark:

File: .github/ISSUE_TEMPLATE/user-story-template.md (hunk 2 above)
Comment (jeremylgn) [reply .github/ISSUE_TEMPLATE/user-story-template.md]: Done :heavy_check_mark:

File: .github/ISSUE_TEMPLATE/user-story-template.md (hunk 3 above)
Comment (jeremylgn) [reply .github/ISSUE_TEMPLATE/user-story-template.md]: Done :heavy_check_mark:

File: .github/ISSUE_TEMPLATE/user-story-template.md (hunk 4 above)
Comment (jeremylgn) [reply .github/ISSUE_TEMPLATE/user-story-template.md]: Done :heavy_check_mark:

File: .github/ISSUE_TEMPLATE/user-story-template.md (hunk 5 above)
Comment (jeremylgn) [reply .github/ISSUE_TEMPLATE/user-story-template.md]: Done :heavy_check_mark:

**Review by ccamel — APPROVED** [review]
:+1:

**Pull request merged**